### DISCUSSION TITLE
How to add a new profile link?

### DISCUSSION THREAD

**Discussion by BenjaminJZubaly** [academicpages/academicpages.github.io#2032] [Q&A]
Hello!

This is probably a very simply thing, but I would like to include my open science framework profile on the side bar along with my other socials. How would I go about doing this, and adding an icon for it as well?

Thanks in advance!!

**Comment by rjzupkoii** [comment]
Here's a walkthrough of how to do it - [How can I add additional social media?](https://github.com/academicpages/academicpages.github.io/wiki/How-can-I-add-additional-social-media%3F) - the OSF icon is part of the Academicons, which should be the only significant change from what's currently written. Let me know if you run into any issues with the walkthrough since I'm trying to make it as comprehensive as possible.

**Reply by BenjaminJZubaly** [reply]
thank you! I don't know how to code other than R (so didn't know what class, etc. meant in the walkthrough), but I got the main gist of it so I ended up just adding the author.osf piece in the _config.yml then, in author-profile.html, copying the html snippet referring to the pubmed Academicon and replacing pubmed with osf.

if I can do it than anyone can, so thanks for the walkthrough. cheers!

**Reply by rjzupkoii** [reply]
Thanks for the feedback - I've clarified the text slightly and and added a link to CSS class information which should help those new to web development a bit.
